CWE/CWE.csproj

```diff
@@ -15,6 +15,10 @@
     <PackageReference Include="Discord.Net.Labs.WebSocket" Version="2.3.8" />
     <PackageReference Include="Discord.Net.WebSocket" Version="2.2.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.5" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="5.0.5">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
Why was this added if there were no adjustments made to the database?

Reply:
would not let me update database on my end in order to test the bot without it

Reply:
This is indeed true. For me to run 'Update-Databade', it was required that the main CWE project had this NuGet package installed.

Reply:
Make sure you set the "Default project" to CWE.Data :)

Reply:
Ah, never thought of that.